server/constants.js

```diff
@@ -31,14 +31,18 @@ module.exports = {
         text: 'Account',
         href: '/account'
       },
-      messages: {
-        text: 'Messages',
-        href: '/messages'
-      },
       data: {
         text: 'Manage Data',
         href: '/data-manage'
       },
+      home: {
```

Comment:
I've added a `Home` item to the navigation menu as it isn't obvious how you would get back.